script/breakupScript.py

```diff
@@ -0,0 +1,65 @@
+import sys
+
+import numpy
+import numpy as np
+import scipy as sp
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import pandas as pd
+
+
+def histogram_lc(data, title, cumulative):
```

Comment:
For nice resolution I would recommend creating a figure first and setting the dpi on it with something like
`fig = plt.figure(figsize=(4,4),dpi=300)`

This create a 4 by 4 inch figure dpi 300. For print 300 or 600 is usually recommended, for presentations etc. 150.